spring-oracle-example/compose.yaml

```
@@ -0,0 +1,32 @@
+
+  oracle-db:
+    image: gvenzl/oracle-xe:21-slim
+    container_name: oracle-xe
+    restart: unless-stopped
+    ports:
+      - "1521:1521"
+    environment:
+      ORACLE_PASSWORD: "Admin123"
+    shm_size: 1g
+    volumes:
+      - ./docker/oracle:/docker-entrypoint-initdb.d
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)